package.json

```diff
@@ -14,6 +14,8 @@
   "author": "OpenZipkin <openzipkin.alt@gmail.com>",
   "repository": "https://github.com/openzipkin/zipkin-js",
   "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-preset-es2015": "^6.18.0",
```

Comment:
Yes, using it from browsers is a good point... in that case it's probably best to do it like you have, use babel-preset-es2015 and let that be used from both browsers and node servers.